README.md

```diff
@@ -173,4 +173,6 @@ To scan a GCP ...
 
 #### Azure
 
-TODO
+Using a computer already configured to use azure-cli, you may use Scout using the following command:
```

Comment:
I don't know if this is possible, but with `azure-cli` can you have multiple accounts configured at once? If so, we'd need a way to tell Scout which configured account to use.

Reply:
Currently looking into the multiple account. This doesn't seem really documented, but the SDK code is open-source and there are comments regarding the arguments.

Reply:
Hi! So for using multiple user accounts, this does not seems to be directly possible, as stated in https://github.com/Azure/azure-xplat-cli/issues/2198. 

However, it may be possible to switch between different subscription, as the azure-cli tool permits it.

Reply:
Could you please look into https://ncc-scout.slack.com/archives/CDCSGNQ8M/p1548008131030700. While using the `azure-cli` seems to be very convenient, it might not be the way to go. One of the primary reasons why I believe ADAL is a better option is that with `azure-cli` it will be hard to run the tool programatically (e.g. spin up a container to run Scout, or run it in lambda, etc. - these are scenarios we are currently using).

It will require a bit more core to use ADAL as you'll have to handle all (4?) authentication scenarios, but it will be beneficial in the long run.